src/test/java/lesson_jpa/OrderDaoImplTest.java

```diff
@@ -1,35 +1,37 @@
 package lesson_jpa;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
 import java.math.BigDecimal;
 import java.sql.Date;
 import java.sql.SQLException;
-import java.util.Set;
 
 import org.junit.FixMethodOrder;
 import org.junit.Test;
 import org.junit.runners.MethodSorters;
 
 @FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class OrderDaoImplTest {
-	private static final BigDecimal NOT_EXIST_ORDER = BigDecimal.valueOf(1111110);
```

Comment:
It is not good. You crash your jdbc examples =(

Reply:
No, I don't crush JDBC, it's just 2 commit for jpa hw...